packages/sources/src/styling/variables/zindex.scss

```diff
@@ -1,5 +1,45 @@
-$zindex-navbar: 1000;
-$zindex-dropdown: 1000;
-$zindex-popover: 1060;
-$zindex-tooltip: 1070;
-$zindex-navbar-fixed: 1030;
+// Selectors
+$zx-viewer-grid-sizing-buttons:     -1;
+$zx-viewer-grid-map-panel:          100;
+$zx-selectable:                     100;
+$zx-description-open-after:         100;
+$zx-datepicker-date-day:            100;
+$zx-datepicker-date-select-label:   100;
+$zx-datepicker-date-select-input:   101;
+$zx-datepicker-date-toggle:         101;
+$zx-description-open-before:        101;
+$zx-viewer-grid-overlay:            101;
+$zx-searchbar-icon:                 101;
+$zx-form-control-feedback:          101;
+$zx-input-group-button:             101;
+$zx-input-group-form-control:       101;
+$zx-input-group-form-control-focus: 102;
+$zx-leaflet-map-pane-canvas:        110;
```

Comment:
Waarom groepeer je dropdowns wel, maar dit leaflet setje niet?
Is dat omdat die andere 'groep' op een compleet andere range zit?